services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -722,6 +722,10 @@ export class WorkflowService {
       documents: workflow?.context?.documents?.map(
         (document: DefaultContextSchema['documents'][number]) => ({
           ...document,
+          decision: {
+            ...document?.decision,
+            status: document?.decision?.status === null ? undefined : document?.decision?.status,
```

Comment:
This is a copy-paste mistake. Should be:
```suggestion
status: document?.decision?.status === null ? undefined : decision?.status,
```